docs/src/administration/configuration.md

```diff
@@ -88,6 +88,14 @@ The following includes ThreadPool related parameters, which can be passed in thr
 - a PKCS#8 private key file in PEM format. e.g. /home/tidb/client-key.pem.
 - default: null
 
+#### tikv.tls.reload_interval
+- The interval in seconds to poll the change of TLS context, if a change detected, the SSL context will be rebuilded.
```

Comment:
if a change detected -> if a change is detected

Reply:
Done.